regression/cbmc/Quantifiers-not-exists/test.desc

```diff
@@ -1,15 +1,7 @@
-KNOWNBUG
+CORE
 main.c
-
-^\*\* Results:$
-^\[main.assertion.5\] line 28 assertion a\[.*\]\[.*\] > 10: SUCCESS$
-^\[main.assertion.6\] line 30 assertion tmp_if_expr\$\d+: SUCCESS$
-^\[main.assertion.7\] line 31 assertion tmp_if_expr\$\d+: SUCCESS$
-^\[main.assertion.8\] line 33 assertion tmp_if_expr\$\d+: SUCCESS$
-^\[main.assertion.9\] line 35 assertion tmp_if_expr\$\d+: SUCCESS$
-^\[main.assertion.10\] line 36 assertion tmp_if_expr\$\d+: SUCCESS$
-^\*\* 0 of 10 failed
-^VERIFICATION SUCCESSFUL$
+--cover location
```

Comment:
At one point there was a PR moving all cover regressions to a directory other than `cbmc/`.

Reply:
Indeed, moved to `cbmc-cover`.

Reply:
Cool.